include/openPMD/Series.hpp

```diff
@@ -346,10 +347,22 @@ class SeriesImpl : public AttributableImpl
     std::unique_ptr< ParsedInput > parseInput(std::string);
     void init(std::shared_ptr< AbstractIOHandler >, std::unique_ptr< ParsedInput >);
     void initDefaults( IterationEncoding );
+    /**
+     * @brief Internal call for flushing a Series.
+     *
+     * Any flushing of the Series will pass through this call.
+     * 
+     * @param begin Start of the range of iterations to flush.
+     * @param end End of the range of iterations to flush.
+     * @param level Flush level, as documented in AbstractIOHandler.hpp.
+     * @param flushIOHandler Tasks will always be enqueued to the backend.
+     *     If this flag is true, tasks will be flushed to the backend.
+     */
     std::future< void > flush_impl(
         iterations_iterator begin,
         iterations_iterator end,
-        FlushLevel );
+        FlushLevel level,
+        bool flushIOHandler );
```

Comment:
Do we want to add `= true` as default parameter, since this is usually what we need and expect?
That was the old behavior and seems logical that `flush_impl()` calls `flush()`?

Reply:
On the one hand, I like to keep those things explicit in internal usage (example: I already had a merge conflict where I had added different flags to the same function, but since those flags had no defaults, git showed me the conflict for each invocation, making things easier to fix). 
However, there's a very clear call for a default in this instance, so I can do a follow-up PR and add it if you like?

Reply:
Sounds good!